bfvmm/src/hve/arch/intel_x64/xen/xen_op.cpp

```diff
@@ -874,7 +1094,6 @@ xen_op_handler::XENVER_get_features_handler(
         info->submap |= (1 << XENFEAT_memory_op_vnode_supported);
         // info->submap |= (1 << XENFEAT_ARM_SMCCC_supported);
         info->submap |= (1 << XENFEAT_linux_rsdp_unrestricted);
-        info->submap |= (1 << XENFEAT_hvm_pirqs);
```

Comment:
are you sure about this one? I would leave this on unless you have a specific reason to turn it off.

Reply:
Yep thats my bad